sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/TrackNotImplementedException.cs

```diff
@@ -29,8 +29,10 @@ void FooBar2()
 //          ^^^^^^^^^
         }
 
-        void NotImplemented() =>
-            throw new NotImplementedException(); // FN
+        void ByExpression() => throw new NotImplementedException(); // Noncompliant
```

Comment:
also, please also add some more UTs with throw expressions , not only from arrow methods

e.g.
```
public bool Foo(int i)
{
  return i == 0 ?? throw new NotImplementedException();
}
```

Reply:
Added two test cases.